front/src/applications/operationalStudies/views/ManageTrainSchedule.tsx

```diff
@@ -119,7 +119,7 @@ const ManageTrainSchedule = ({ trainIdToEdit }: ManageTrainScheduleProps) => {
         <div className="floating-itinerary">
           <Itinerary pathProperties={pathProperties} setPathProperties={setPathProperties} />
         </div>
-        <Map pathProperties={pathProperties}>
+        <Map pathProperties={pathProperties} pathGeometry={pathProperties.geometry}>
```

Comment:
I find it pretty weird to pass two times the same data in the `Map` component. `geometry` is already included in `pathProperties` so we shouldn't have to pass it another time in the `pathGeometry` prop. And it the `Map component` we have to sources of truth, it might no be very good.

Reply:
I have changed a bit the behaviour, now you can pass the path geometry only, the path properties or both :+1:

Reply:
Ok. As you spotted it, we could add a small refacto issue to pass only `{ pathGeometry?, pathLength? }` instead of keeping both these props and change pathProperties to this new object until the store.

Reply:
Maybe we can do this in a later PR ? :)

Reply:
Yes of course, I was just suggesting to create the issue so we don't forget :)

Reply:
Here is the issue :) https://github.com/OpenRailAssociation/osrd/issues/9266
I also added to the board of the enhancements